annotations.md

```diff
@@ -22,4 +22,7 @@ This specification defines the following annotation keys, intended for but not l
 * **org.opencontainers.image.homepage** URL to find more information on the image (string, a URL with scheme HTTP or HTTPS)
 * **org.opencontainers.image.documentation** URL to get documentation on the image (string, a URL with scheme HTTP or HTTPS)
 * **org.opencontainers.image.source** URL to get source code for the binary files in the image (string, a URL with scheme HTTP or HTTPS)
+* **org.opencontainers.image.version** [Semantic versioning-compatible](http://semver.org/) version of the packaged software.
+* **org.opencontainers.image.revision** Source control revision identifier for packaged software.
```

Comment:
Yes, but implementations needs to invoke HTTP request for determining the VCS, if we suppose `.revision` to be machine-readable.

So, if we want to use `.source` for this purpose, I think we need to allow `git://` scheme for `.source`.